response_operations_ui/common/token_decoder.py

```diff
@@ -14,6 +14,7 @@ def decode_access_token(access_token):
     decoded_jwt = jwt.decode(
         access_token,
         key=uaa_public_key,
+        algorithms=['RS256'],
```

Comment:
Is this actually a change of algo, or just affirming the use of a different one?

If it's a change, what were we using before?  I assume we went with RS because it's asymmetric?

Reply:
This isn't a change of algo.  This change came from a deprecation warning in pyjwt that in a future version, it won't be possible to not define an algorithm when encoding/decoding (you only need to specify when you encode currently I think..)

This change is just explicitly saying what we have always been using.  I tested this in 2 ways, first looking at the payload we get back from uaa when we request the public key, as it tells us the algorithm used.  Second, I added a print statement to my local copy of pyjwt (roughly around https://github.com/jpadilla/pyjwt/blob/master/jwt/api_jws.py#L213 ) to see what it was using.  Originally it was using RS256 when running the service and HS256 in the unit tests, but now it uses RS256 for both.

Reply:
Seems like a good plan - certainly better than using a different algo in each.  RS is better for this too.

Reply:
This should be HS256 as that is what was always used to be, we also use HS256 in frontstage